vagrant/provision-master.sh

```diff
@@ -17,6 +17,9 @@ for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
     echo "$ip $minion" >> /etc/hosts
   fi
 done
+if [ ! "$(cat /etc/hosts | grep ${MASTER_IP})" ]; then
```

Comment:
Why cat and grep vs just grep?
and why check for "" vs checking for an exit code?